src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/GraphNodes/DependenciesGraphChangeTracker.cs

```diff
@@ -150,7 +152,7 @@ bool HandleChanges(IGraphContext graphContext)
                         nodeProjectPath,
                         updatedDependency,
                         inputGraphNode,
-                        updatedSnapshot.Targets[updatedDependency.TargetFramework]))
+                        updatedSnapshot!.Targets[updatedDependency.TargetFramework]))
```

Comment:
Asking for forgiveness with `!` here to work around https://github.com/dotnet/roslyn/issues/36014

Reply:
Might be worth a comment in the code here indicating that this is okay because of the early return from `updatedDependency == null`, as that wouldn't have been apparent to me if I hadn't just read that roslyn issue (plus it doesn't sound like this is a temporary workaround)